decode.c

```diff
@@ -91,6 +91,7 @@ API_SYM unsigned x86emu_run(x86emu_t *emu, unsigned flags)
 
     if(ACC_D(emu->x86.R_CS_ACC)) {
       emu->x86.mode |= _MODE_DATA32 | _MODE_ADDR32 | _MODE_CODE32;
+	  emu->x86.mode |= _MODE_STACK32; // 2020-05-31 to be reviewed
```

Comment:
No, that's not correct. The CS selector's D bit provides only the default for code and ('normal') data accesses - in absense of a segment prefix. For stack accesses always the SS segment info is used. IIRC I did some experiments
on real hardware to get this implemented correctly back then.

I might be wrong, though. So, if you have a small code blob that works differently than the emulation, please post it here.